<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ball Bearing Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="B_Height">
     <property name="specialValueText">
      <string/>
     </property>
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>7.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Internal Diameter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="Int_Diam">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>External Diameter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="Ext_Diam">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>22.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Minimum Wall Thickness</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="Min_Wall_Thick">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Ball Size Rounding</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="Ball_Rounding">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
